### PR TITLE
#47 Fixed Calendar entries - make end date update to the start date

### PR DESCRIPTION
**Pull request by MartinMaritim**
Hi @guillaumev. Kindly have a look, I tested this on my local dev and works well. Cheers,

### CODE REVIEW

**Comment by guillaumev** [comment]
Working for the date field but not for the time field...

**Comment by MartinMaritim** [comment]
@guillaumev will work on it ...

**Comment by MartinMaritim** [comment]
@guillaumev working well for the time field now ...

File: sites/all/modules/hr/hr_events/js/hr_events.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+(function($) {
+  Drupal.behaviors.hrEventsDatePopupValue = {
+    attach: function (context, settings) {
+      $('#edit-field-event-date-und-0-value-datepicker-popup-0', context).on('change', function(){ 
+          $('#edit-field-event-date-und-0-value2-datepicker-popup-0').val($(this).val());
+      });
+<<<<<<< HEAD
```
Comment (pcambra) [inline sites/all/modules/hr/hr_events/js/hr_events.js]: Are these leftovers from a merge?

File: sites/all/modules/hr/hr_events/js/hr_events.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      $('#edit-field-event-date-und-0-value-timeEntry-popup-1', context).on('change', function(){ 
+          $('#edit-field-event-date-und-0-value2-timeEntry-popup-1').val($(this).val());
+      });
+=======
+>>>>>>> 74ff28a2f8e5ae55e3466fb8d8b038bac594de4a
```
Comment (pcambra) [inline sites/all/modules/hr/hr_events/js/hr_events.js]: Here as well

Comment (MartinMaritim) [inline sites/all/modules/hr/hr_events/js/hr_events.js]: @pcambra I was fixing a git confict, which was resolved by adding the changed file to the staging area, committing and pushing it to the repo.

Comment (MartinMaritim) [inline sites/all/modules/hr/hr_events/js/hr_events.js]: @pcambra I can revert the commit, do a clean push. Thoughts?

Comment (pcambra) [inline sites/all/modules/hr/hr_events/js/hr_events.js]: You could try amend as well, https://www.atlassian.com/git/tutorials/rewriting-history/git-commit--amend/ or doing a rebase, let me know if you need help

Comment (MartinMaritim) [inline sites/all/modules/hr/hr_events/js/hr_events.js]: Ok thanks @pcambra. Will work on it ...

**Comment by MartinMaritim** [comment]
@pcambra Kindly have a look, I hope we now have something good to push to live :)

**Comment by guillaumev** [comment]
I'll let @pcambra merge it but this looks good to me

**Pull request merged**